lib/awful/widget/prompt.lua

```diff
@@ -28,6 +29,9 @@ local function run(promptbox)
         exe_callback        = function (...)
             promptbox:spawn_and_handle_error(...)
         end,
+        fg_cursor           = beautiful.prompt_fg_cursor,
+        bg_cursor           = beautiful.prompt_bg_cursor,
+        font                = beautiful.prompt_font,
```

Comment:
I kept them here because foreground and background color are set here as well. If people don't want to use this wrapper they can directly use `awful.prompt`.